File: features/0 deleting stuff/delete.js
exports.run = (msg,client) => {
  msg.formatted = client.misc.polishchars(` ${msg.content} `).toLowerCase().replace(/["'\*\.]/g,"");
  if(!msg.formatted.includes(atob('a3J6eXM=')))return;
  client.misc.log(msg.content);
  let out = 'stop';
  msg.delete().catch(e=>{});
  return out;
}

File: features/0_deleting stuff/delete.py
import base64
import re

def run(msg, client):
    """Polish message for readability by the bot\n
    And then verify krzys isn't in it"""
    msg.formatted = re.compile("[\"\'*.]").sub(client.misc.polishchars(f"{msg.content}").lower(), "")
    #if not "krzys" in msg.formatted:										# Exchange the place of # to
    if not base64.b64decode('a3J6eXM=').decode('utf-8') in msg.formatted:	# change between slow and fast way
        return
    client.misc.log(msg.content)
    out = 'stop'
    try:
        msg.delete()
    except:
        pass
    return out